src/libraries/System.Text.RegularExpressions/tests/Regex.Groups.Tests.cs

```diff
@@ -867,6 +867,7 @@ public static IEnumerable<object[]> Groups_CustomCulture_TestData_AzeriLatin()
         [MemberData(nameof(Groups_CustomCulture_TestData_Danish))]
         [MemberData(nameof(Groups_CustomCulture_TestData_Turkish))]
         [MemberData(nameof(Groups_CustomCulture_TestData_AzeriLatin))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/39528", TestPlatforms.Browser)]
```

Comment:
I think this is because of the locales we support in WASM. See: https://github.com/dotnet/runtime/issues/39528#issuecomment-660263964

Reply:
I see, so that test case can just be skipped for browser via `if (PlatformDetection.IsNotBrowser)`?

Reply:
I think so. I'll let @EgorBo confirm.

Reply:
I think you can allow everything but [MemberData(nameof(Groups_CustomCulture_TestData_AzeriLatin))] now